include/boost/url/url.hpp

```diff
@@ -905,6 +946,10 @@ class BOOST_SYMBOL_VISIBLE url
     url&
     set_port(string_view s);
```

Comment:
This string can contain only digits

Reply:
See https://github.com/CPPAlliance/url/pull/238#discussion_r927276255